script/munin-doc

```diff
@@ -32,55 +32,93 @@ use Pod::Perldoc;
 use File::Find;
 use Munin::Common::Defaults;
 
-my @found = ();		# Found plugin path names, by priority
+local $ENV{PATH} = '/usr/bin/local:/usr/bin:/bin';
 
-my @myargv = @ARGV;	# Save the ARGV we want for ourselves.
+my @found = ();    # Found plugin path names, by priority
 
-my ($plugin) = @myargv; # First argument is a plugin name
+my @myargv = @ARGV;    # Save the ARGV we want for ourselves.
+
+my ($plugin) = @myargv;    # First argument is a plugin name
 
 die "munindoc: Please name a plugin\n" unless $plugin;
 
+# un-taint plugin name
+die "munindoc: Invalid plugin name '$plugin'\n" unless $plugin =~ /^([\w-]+)$/x;
+$plugin = $1;
+
 my $plugin_re = quotemeta($plugin);
 
-@ARGV=();
+local @ARGV = ();
 
 # Dirs in which to look for $plugin.pod and $plugin files.
-my @DIRS = ("$Munin::Common::Defaults::MUNIN_LIBDIR/plugins",
-	    "$Munin::Common::Defaults::MUNIN_CONFDIR/plugins");
+my @DIRS = (
+    "$Munin::Common::Defaults::MUNIN_LIBDIR/plugins",
+    "$Munin::Common::Defaults::MUNIN_CONFDIR/plugins"
+);
+
+File::Find::find( { wanted => \&wanted_pod,      untaint => 1 }, @DIRS );
+File::Find::find( { wanted => \&wanted_basename, untaint => 1 }, @DIRS );
 
-File::Find::find({wanted => \&wanted_pod}, @DIRS);
-File::Find::find({wanted => \&wanted_basename}, @DIRS);
+die "munin-doc: Plugin '$plugin' not found\n" if !@found;
 
-# print "Found: ",join(", ",@found),"\n";
-# exit 0;
+my ($found_first) = @found;
 
 # -F  Arguments are file names, not modules
-push(@ARGV,'-F',@found);
+push( @ARGV, '-F', $found_first );
```

Comment:
It's a change from perltidy (probably http://perltidy.sourceforge.net/stylekey.html -> *Define Horizontal Tightness*).
I think it can be changed by `-pt=2`.
It's up to the two of you to decide :)

Reply:
@steveschnepp: what do you think?

(my python-infested taste would prefer to avoid whitespace here)